src/containers/Application.js

```diff
@@ -446,23 +440,13 @@ class Application extends Component {
 		}
 	}
 
-	// First argument is href to page
-	// Second (optional) is string that is icon from core-ui icons
-	// Third (optional) is target for link
-	addHelpButton(url, icon = "cil-info", target) {
+	addHelpButton(path) {
 		useEffect(() => {
-			this.Store.dispatch({
-				type: CHANGE_HELP_URL,
-				payload: { url, icon, target }
-			})
+			this.HelpService.setData(path);
 			return () => {
 				this.Store.dispatch({
-					type: CHANGE_HELP_URL,
-					payload: {
-						url: this.Config.get('default_help_url'),
-						icon: "cil-info",
-						target: "_blank"
-					}
+					type: HELP_DESCRIPTION,
+					description: ""
```

Comment:
@Aringoaway As my comment above - I think it should be `content` rather than `description`

Reply:
@Aringoaway Maybe it was misunderstanding, but I thought that you will rewrite `description` to `content` as well, since it makes no sense to me to have a `description` here (and elsewhere)